### ISSUE TITLE
accessibility vs a11y query parameter

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/chipper#716]
Based on Slack conversation:

- `?accessibility` is intended for internal use, not public facing
- some team members (myself, @arouinfar, others?) typically try `?a11y`

Would it make sense to change the query parameter to 'a11y'? Or provide 'a11y' as an alias?

**Comment by jessegreenberg** [comment]
I added a11y as an alias for the accessibility query parameter. The Sim.js and Display.js options to enable are also called `accessibility`, so I think it is good to keep a query parameter with the same name. But an alias will be very helpful. Closing this issue.
